extensions/default/src/DicomLocalDataSource/index.js

```diff
@@ -221,7 +221,8 @@ function createDicomLocalApi(dicomLocalConfig) {
       //   return instance.imageId;
       // }
 
-      const { StudyInstanceUID, SeriesInstanceUID, SOPInstanceUID } = instance;
+      const { StudyInstanceUID, SeriesInstanceUID } = instance;
+      const SOPInstanceUID = instance.SOPInstanceUID || instance.SopInstanceUID;
```

Comment:
this is very odd, i did not expect this to be an issue, is there really two things on the instance?

Reply:
it can be seen here too, so I guess its a thing https://github.com/cornerstonejs/cornerstone3D/blob/61c7d8beff128eed837feacdda495c9ec21780c2/packages/adapters/src/adapters/Cornerstone3D/Segmentation/generateSegmentation.ts#L45